src/NUnitFramework/framework/Internal/Execution/EventListenerTextWriter.cs

```diff
@@ -68,10 +68,8 @@ private bool TrySendToListener(string text)
             if (context == null || context.Listener == null)
                 return false;
 
-            string testName = context.CurrentTest != null
-                ? context.CurrentTest.FullName
-                : null;
-            context.Listener.TestOutput(new TestOutput(text, _streamName, testName));
+            var test = context.CurrentTest ?? null;
```

Comment:
Can you remove the `?? null` since it has no effect? It might even make sense to inline the `test` variable.

Reply:
Nice catch ;)